config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+import (
+	"regexp"
+
+	"github.com/elastic/apm-server/sourcemap"
+)
+
+type Config struct {
```

Comment:
we have so many configs that i would prefer to have more explicit names for each one, like eg. `FrontendConfig` for this one, but up to you

Reply:
I would not change it to `FrontendConfig` with this PR as this was only meant to get rid of package dependencies.
AFAIR the `LibraryPattern` and `ExcludeFromGrouping` could theoretically also be used for backend endpoints as discussed when we introduced these settings, so I would revisit this in a separate PR.